apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -57,11 +60,14 @@ function getSteps(template: NotificationTemplateEntity, controlValuesMap: { [p:
 }
 
 function toMinifiedWorkflowDto(template: NotificationTemplateEntity): WorkflowListResponseDto {
+  const workflowName = template.name || 'Missing Name';
+
   return {
+    _id: template._id,
+    slug: `${slugifyName(workflowName)}_${encodeBase62(template._id)}`,
```

Comment:
self note: add the entity name  `-stp-`  before the id so that it is easy to support backward compatibility with these ids once we shorten them

Reply:
same here

remove slugifyName(workflowName), slug should be `{prefix}{shortId}` in order to support future refactoring, the client should structure the actual slug with {name}{slug}